rj_gameplay/rj_gameplay/skill/intercept.py

```diff
@@ -5,57 +5,82 @@
 import py_trees
 import sys
 import time
+import numpy as np
+from rj_msgs.msg import RobotIntent, PathTargetMotionCommand
+from rj_geometry_msgs.msg import Point
 
 import stp.skill as skill
 import stp.role as role
-import stp.action as action
-from rj_gameplay.action import intercept
-from stp.skill.action_behavior import ActionBehavior
 import stp.rc as rc
-import numpy as np
+from typing import Optional
 
+SETTLE_BALL_SPEED_THRESHOLD = 1.0
+INTERCEPT_ANGLE_THRESHOLD = 2*np.pi / 36
 
-class IIntercept(skill.ISkill, ABC):
-    ...
 
+class Intercept(skill.Skill):
+    """First third of a PassReceive Skill. Gets in front of moving ball."""
+    """TODO: Replace with actual intercept planner"""
 
-"""
-A skill version of intercept so that actions don't have to be called in tactics
-"""
+    def __init__(self, robot: rc.Robot = None):
+        self.robot = robot
 
+        self.__name__ = "intercept skill"
 
-# TODO: discuss collapsing skills/actions
-class Intercept(IIntercept):
-    def __init__(
-        self,
-        robot: rc.Robot = None,
-        target_point: np.ndarray = np.array([0.0, 0.0]),
-    ):
-        self.robot = robot
-        self.target_point = target_point
-
-        self.__name__ = "Intercept Skill"
-        if self.robot is not None:
-            self.intercept = intercept.Intercept(self.robot.id, target_point)
-        else:
-            self.intercept = intercept.Intercept(None, target_point)
-        self.intercept_behavior = ActionBehavior(
-            "Intercept", self.intercept, self.robot
+    def get_intercept_pt(self, world_state: rc.WorldState, my_robot: np.ndarray) -> np.ndarray:
```

Comment:
Can you add a docstring here explaining what this does? Fairly certain it's "find point perp to the ball-bot line" but some clarification would be good for posterity